index.html

```diff
@@ -5,8 +5,16 @@
 	<title>
 		beautiful and simple website using html only -- fullywrold web tutorials
 	</title>
+	<style>
+		.bg{
+			background-image: url('1.jpg');
+			background-position: center;
+			background-repeat: repeat-y;
+			background-size: 100%;
+		}
+	</style>
 </head>

```

Comment:
Please Create Another HTML Page with its UI/UX similar to the main one & submit you PR. Otherwise such minor changes will not be accepted. You can add projects.html etc